### PR TITLE
If on MacOS use code.app as the binary name

### PR DESCRIPTION
**Pull request by thibaudio**

### CODE REVIEW

**Comment by elizagamedev** [comment]
Thanks for the contribution! Would you mind revising to use `(cond)` instead of the if else chain here?

**Comment by thibaudio** [comment]
Thanks for the feedback!
I've updated the PR

**Comment by GeorgesAlkhouri** [comment]
Is there something left to do here or could this be merged?

File: unity.el [hunk 1]
```diff
@@ -88,11 +88,9 @@ FILE and TRASH are documented by `rename-file'."
...
 (defun unity--code-binary-file ()
   "Return the file name of the code shim binary."
-  (concat unity-var-directory (if (eq system-type 'windows-nt)
-                                  "code.exe"
-                                (if (eq system-type 'darwin)
-                                    "code.app"
-                                "code"))))
+  (concat unity-var-directory (cond ((eq system-type 'windows-nt) "code.exe")
+                                    ((eq system-type 'darwin) "code.app")
+                                    "code")))
```
Comment (elizagamedev) [inline unity.el]: This is incorrect--this line should be (t "code")))

**Comment by elizagamedev** [comment]
Sorry, this slipped through the cracks. The syntax for `cond` is incorrect, but once it's fixed I can merge.

**Comment by thibaudio** [comment]
Sorry, missed this one.
I've updated the syntax !

**Pull request merged**